.github/workflows/ci-core.yml

```diff
@@ -10,12 +10,11 @@ on:
 
 jobs:
   test:
+    runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu ]
         python-version: ["3.9", "3.10", "3.11", "3.12"]
-    runs-on: ${{ matrix.os }}-latest
     steps:
       - uses: actions/checkout@v4
       - name: Set up Python
```

Comment:
i think this is also aspirationally there for win/mac OSs, but again they are missing as a cost saving measure.

Reply:
@alexanderankin Please be aware of the findings from @bearrito and me concerning ARM runners in this thread: https://github.com/testcontainers/testcontainers-python/issues/446

It looks like these runners are usable mid or long term. Maybe end of this year. We don't know.

Currently this matrix does not serve any purpose. So I consider this as dead code which can be removed.